lib/tty/prompt/multi_list.rb

```diff
@@ -89,6 +133,30 @@ def keyctrl_r(*)
 
       private
 
+      # Initialize any default or custom select keys
+      # setting up their labels and dealing with any key conflicts
+      #
+      # @see List#init_action_keys
+      #
+      # @api private
+      def init_select_keys(keys)
+        @select_keys = init_action_keys(keys)
+        check_conflicting_keys
+        @select_keys
+      end
+
+      # Checks that there are no key options clashing
+      #
+      # @api private
+      def check_conflicting_keys
+        conflicting_keys = @confirm_keys.keys & @select_keys.keys
+        return if conflicting_keys.empty?
+
+        raise ConfigurationError,
+              ":confirm_keys #{conflicting_keys} are conflicting with " \
```

Comment:
I have a proposal for a slightly modified version of this error message:

```ruby
":confirm_keys and :select_keys cannot use the same " \ 
"#{conflicting_keys.map(&:inspect).join(", ")} " \
"key#{"s" if conflicting_keys.size > 1}" 
```

What do you think?